src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -25,8 +30,21 @@
 	private static void setupPresetTests(Application application) {
 		SelectTestFigureOptionListener selectTestFigureOptionListener = new SelectTestFigureOptionListener(
 				DriverFeature.getDriverManager());
-
+		SelectSecondTestFigureOptionListener select = new SelectSecondTestFigureOptionListener((
+				DriverFeature.getDriverManager()
+				));
+		
 		application.addTest("Figure Joe 1", selectTestFigureOptionListener);
+		application.addTest("Figure Joe 2", select);
+		
+		application.addTest("Figure Jane 1", new ActionListener()
```

Comment:
It is good to keep a convention used in the original code (class instance instead of lambda)